Lib/shutil.py

```diff
@@ -170,6 +171,35 @@ def _copyxattr(src, dst, *, follow_symlinks=True):
     def _copyxattr(*args, **kwargs):
         pass
 
+if sys.platform == 'darwin':
+    # As of macOS 10.10, the `copyfile` API changed to not copy certain flags
+    mac_ver = tuple(int(x) for x in platform.mac_ver()[0].split('.'))
+    if mac_ver >= (10, 10, 0):
+        def _fix_flags(dst, flags, follow_symlinks):
+            """Perform any modifications to `flags` before they can be applied
+            to `dst` with `chflags()`.
+
+            """
+
+            # From xnu's bsd/sys/stat.h
+            UF_TRACKED    = 0x00000040
+            SF_RESTRICTED = 0x00080000
```

Comment:
These constants need to be to be added to Modules/posixmodule.c (like other related constants already are). Although this is problematic for the python.org installers as those are build on a 10.9 system and probably won't pick up these definitions when there are added to posixmodule.c.

Reply:
The related constants are in Lib/stat.py, not in posixmodule.c. Which is correct?